src/api/mdn.js

```diff
@@ -117,22 +143,28 @@ function replacePlaceholders(string, replacements) {
 /**
  * Accepts a relative URL or an attachment object
  * Returns the content of a given file.
+ *
+ * @param {string|{url:string}} fileObjOrUrl
  */
 async function getFileContent(fileObjOrUrl) {
-    var url = fileObjOrUrl.url || fileObjOrUrl;
-    if (!url) return '';
+    /** @type {string} */
+    // @ts-ignore: Type unsafe parameter detection
+    var file_url = fileObjOrUrl.url || fileObjOrUrl;
+    if (!file_url) return '';
 
     let base_url = '';
 
-    // New file urls include attachment host, so we don't need to
-    // prepend it
-    var fUrl = url.parse(url);
+    /**
+     * New file urls include attachment host, so we don't need to
+     * prepend it
+     */
+    var fUrl = url.parse(file_url);
```

Comment:
Previously, `url` would’ve been a `string` due to it being shadowed by the `file_url` variable, which broke `Embed_Text`: https://github.com/mdn/kumascript/pull/1047#discussion_r270607750